pkg/containertools/dockerfilegenerator.go

```diff
@@ -8,10 +8,11 @@ import (
 )
 
 const (
-	DefaultBinarySourceImage = "quay.io/operator-framework/opm:latest"
-	DefaultDbLocation        = "/database/index.db"
-	DbLocationLabel          = "operators.operatorframework.io.index.database.v1"
-	ConfigsLocationLabel     = "operators.operatorframework.io.index.configs.v1"
+	DefaultScratchSourceImage = "scratch"
```

Comment:
I don't think there's any such this as "a different scratch image", so the naming of this feels weird.

I think I'd suggest one of:
- no constant at all
- an unexported constant in `cmd/opm/generate/cmd.go`, but named simply `scratchImage` or similar.

Reply:
I was originally using this in cmd as well as actions, so a constant where existing constants were made sense.  However, the template changes necessitate a bare string comparison, and evidently the utest uses got lost too, leaving only the one use in cmd... and this is not my hill to die on. ;)